scripts/main.py

```diff
@@ -0,0 +1,150 @@
+import os
+from dotenv import load_dotenv
+import tweepy as tw
+import re
+import string
+import nltk
+import pandas as pd
+from sklearn.cluster import DBSCAN
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+# NLTK variables to aid in cleaning/preprocessing
+ps = nltk.PorterStemmer()
+
+# Pandas dataframe for tweets
+df = pd.DataFrame(columns=["id", "tweet", "popularity"])
+
+
+class Results:
+    def __init__(self):
+        """Initialize final results of the analysis"""
+        self.clusters_count = df.clusters.value_counts()
+        self.df_results = df.groupby(["clusters"]).max().reset_index()
+        print("Number of tweets per cluster: \n{}".format(self.clusters_count))
```

Comment:
Try using `f string` while printing.

eg.,
> print(f"Number of tweets per cluster: {self.clusters_count}")

#### Better Option
instead of print, we can use `logging` in python or `loguru` to have this information inside the log file.

Reply:
Alright, I will actually just use logging.